### PR TITLE
chore: remove write_to_firestore

### PR DESCRIPTION
**Pull request by mederka**
## Description
Removes `write_to_firestore` magic function

### Integration tests

Adapter to test:
- postgres
- bigquery

Python version to test:

- 3.8
- 3.9

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -36,11 +36,6 @@ virtualenv = "^20.16.2"
 dill = "^0.3.5.1, <0.3.6"
 importlib-metadata = "^4.12.0"
 
-# TODO: review any of these are bigquery-specific or for firebase
-google-cloud-bigquery = "^2.28.1"
-google-cloud-bigquery-storage = "^2.9.1"
-firebase-admin = "^5.1.0"
```
Comment (chamini2) [inline pyproject.toml:42]: this does not break dbt-bigquery support?

**Review by chamini2 — COMMENTED** [review]
Curious why tests did not run, let's see bigquery results.

File: pyproject.toml [hunk 2]
```diff
@@ -19,7 +19,7 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.7.2, <3.11"
+python = ">=3.7.2, <4.0"
```
[the remark this answers is not in the document]
Comment (mederka) [reply pyproject.toml:22]: Yep. Why not? 🤠

**Comment by mederka** [comment]
Tests ran earlier before the action that updated the lockfile: https://github.com/fal-ai/fal/actions/runs/3901548260

File: pyproject.toml (hunk 1 above)
Comment (mederka) [reply pyproject.toml:42]: Nope, here are test results: https://github.com/fal-ai/fal/actions/runs/3901548260

**Review by chamini2 — APPROVED** [review]
Awesome, thanks!

**Pull request merged**